tests/behat/features/create-taxonomies.feature

```diff
@@ -38,6 +38,6 @@ Feature: Create taxonomies
     And I press the "Create" button
     When I follow "My taxonomy term"
     And I click the "Children" CMS tab
-    Then I should see "My child taxonomy term"
+    Then I should see "My child taxonomy term" in the "#Form_ItemEditForm_Children" element
     When I follow "Taxonomy Terms"
-    Then I should not see "My child taxonomy term"
+    Then I should not see "My child taxonomy term" in the "#Form_EditForm_SilverStripe-Taxonomy-TaxonomyTerm" element
```

Comment:
```suggestion
    # This needs a different selector, because it's in a different tab
    Then I should not see "My child taxonomy term" in the "#Form_EditForm_SilverStripe-Taxonomy-TaxonomyTerm" element
```

Reply:
Updated